pkg/pillar/zedcloud/authen.go

```diff
@@ -511,6 +512,20 @@ func VerifySigningCertChain(log *base.LogObject, certs []*zcert.ZCert) ([]byte,
 			}
 		}
 	}
+
+	// verify date range of certificates
+	for _, cert := range certs {
```

Comment:
this function gets called very time we receives a message from the cloud, to check the validity on every time is not necessary. have a way to check say once a day would be good.

Reply:
+1
I don't know if it is better having a separate timer for this, or just have this function track the last time it checked the certificates.